### DISCUSSION TITLE
Remap <ESC> to norma mode issue

### DISCUSSION THREAD

**Discussion by membranobruno** [NvChad/NvChad#2192] [General]
Hello everybody! I  was using Sublime Text with vintage mode for a long time. And I'm used to "df" shortcut instead of \<esc\> to escape the insert mode. So I put the next line to  mapping.lua

["df"] = {"\<esc\>", "Escape insert mode", opts = { silent = true }}

It works well if I type "df" (it turns me to Normal mode) and if I type "d" and any letter after shortly, I just continue the typing.
But if my word ends with "d", for example "word" and nothing after the "d" letter the cursor jumps to the bottom panel and wait for the "f" ))) And I have no idea how to break this behaviour. And I don’t want to train my fingers to the \<esc\> again )

Thanks for any help.

**Comment by benzwt** [comment]
try this 

remove  
```
["df"] = {"<esc>", "Escape insert mode", opts = { silent = true }}
```
from mapping.

Then, install this plugin
* https://github.com/max397574/better-escape.nvim

**Reply by antoineco** [reply]
In case you aren't familiar with Neovim and NvChad yet, here is how to achieve that through your `custom/plugins.lua` file:

```lua
local plugins = {
  {
    "max397574/better-escape.nvim",
    event = "InsertEnter",
    opts = {
      mapping = { "df" },
    },
  },
}

return plugins
```

**Comment by membranobruno** [comment]
Works like a charm! Thanks a lot
